spec/models/class_import_spec.rb

```diff
@@ -152,7 +150,12 @@
     let(:file) { "valid.csv" }
 
     context "when import_search_pds flag is enabled" do
-      before { Flipper.enable(:import_search_pds) }
+      before do
+        Flipper.enable(:import_search_pds)
+        allow(ProcessPatientChangesetJob).to receive(:perform_later)
+        allow(CommitPatientChangesetsJob).to receive(:perform_later)
```

Comment:
Similarly to my other comment, maybe it's better to expect `PDSCascadingSearchJob.perform_later` to be called.